localstack/services/events/scheduler.py

```diff
@@ -37,11 +38,20 @@ def __init__(self):
         self.jobs = []
         self.thread = None
 
-    def add_job(self, job_func, schedule):
-        job = Job(job_func, schedule)
+    def add_job(self, job_func, schedule, enabled):
+        job = Job(job_func, schedule, enabled)
         self.jobs.append(job)
         return job.job_id
 
+    def disable_job(self, job_id):
+        i = 0
```

Comment:
nit: we could use a `foreach` loop instead of `while` loop with counter here:
```
        for job in self.jobs:
            if job.job_id == job_id:
                job.is_enabled = False
                break
```